### PR TITLE
Unfuck dockerfile

### PR DESCRIPTION
**Pull request by jswny**
@cooperpellaton stop copying stupid code you find online

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,73 +1,36 @@
...
-	# it gets named `.venv`
-	POETRY_VIRTUALENVS_IN_PROJECT=true \
-	# do not ask any interactive question
-	POETRY_NO_INTERACTION=1 \
-	\
-	# paths
-	# this is where our requirements + virtual environment will live
-	PYSETUP_PATH="/opt/pysetup" \
-	VENV_PATH="/opt/pysetup/.venv"
+ARG BUILD_ESSENATIAL_VERSION='12.6*'
```
Comment (ironchefpython) [inline Dockerfile:3]: Don't do this

Comment (jswny) [reply Dockerfile:3]: It's literally the recommended way to do it by Docker themselves so shut

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#general-guidelines-and-recommendations

Comment (ironchefpython) [reply Dockerfile:3]: No

Comment (jswny) [reply Dockerfile]: Removed

**Review by ironchefpython — APPROVED** [review]
Fine

**Pull request merged**